<ui version="4.0" >
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1040</width>
    <height>714</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize" >
   <size>
    <width>600</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>InterVerdiKom [*]</string>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <property name="sizePolicy" >
    <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
     <horstretch>1</horstretch>
     <verstretch>1</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" >
    <item>
     <widget class="QSplitter" name="splitter_2" >
      <property name="orientation" >
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QSplitter" name="splitter" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
         <horstretch>0</horstretch>
         <verstretch>4</verstretch>
        </sizepolicy>
       </property>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QWidget" name="layoutWidget" >
        <layout class="QVBoxLayout" >
         <item>
          <layout class="QHBoxLayout" >
           <item>
            <widget class="QComboBox" name="rootCombo" >
             <property name="enabled" >
              <bool>false</bool>
             </property>
             <property name="sizePolicy" >
              <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
               <horstretch>0</horstretch>
               <verstretch>5</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="ModuleList" name="moduleSelect" >
           <property name="enabled" >
            <bool>true</bool>
           </property>
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Preferred" hsizetype="MinimumExpanding" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize" >
            <size>
             <width>100</width>
             <height>0</height>
            </size>
           </property>
           <property name="baseSize" >
            <size>
             <width>200</width>
             <height>0</height>
            </size>
           </property>
           <property name="verticalScrollBarPolicy" >
            <enum>Qt::ScrollBarAsNeeded</enum>
           </property>
           <property name="editTriggers" >
            <set>QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed|QAbstractItemView::SelectedClicked</set>
           </property>
           <property name="rootIsDecorated" >
            <bool>false</bool>
           </property>
           <property name="uniformRowHeights" >
            <bool>true</bool>
           </property>
           <property name="itemsExpandable" >
            <bool>true</bool>
           </property>
           <property name="animated" >
            <bool>false</bool>
           </property>
           <property name="columnCount" >
            <number>1</number>
           </property>
           <column>
            <property name="text" >
             <string>Modules</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget" >
        <layout class="QVBoxLayout" >
         <property name="spacing" >
          <number>6</number>
         </property>
         <property name="margin" >
          <number>0</number>
         </property>
         <item>
          <widget class="NavigationPath" native="1" name="navigationPath" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize" >
            <size>
             <width>0</width>
             <height>20</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="moduleTitleLabel" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
             <horstretch>249</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize" >
            <size>
             <width>100</width>
             <height>0</height>
            </size>
           </property>
           <property name="styleSheet" >
            <string notr="true" >font:bold 20px;</string>
           </property>
           <property name="text" >
            <string>InterVerdiKom</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="moduleContentFrame" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
             <horstretch>255</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize" >
            <size>
             <width>450</width>
             <height>16</height>
            </size>
           </property>
           <property name="baseSize" >
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="frameShape" >
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow" >
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QGridLayout" >
            <property name="margin" >
             <number>0</number>
            </property>
            <property name="spacing" >
             <number>0</number>
            </property>
            <item row="0" column="0" >
             <widget class="QStackedWidget" name="moduleContent" >
              <property name="sizePolicy" >
               <sizepolicy vsizetype="MinimumExpanding" hsizetype="Expanding" >
                <horstretch>5</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="frameShape" >
               <enum>QFrame::NoFrame</enum>
              </property>
              <property name="frameShadow" >
               <enum>QFrame::Plain</enum>
              </property>
              <property name="lineWidth" >
               <number>0</number>
              </property>
              <property name="midLineWidth" >
               <number>0</number>
              </property>
              <property name="currentIndex" >
               <number>0</number>
              </property>
              <widget class="QWidget" name="welcomePage" >
               <layout class="QVBoxLayout" >
                <property name="spacing" >
                 <number>6</number>
                </property>
                <property name="margin" >
                 <number>9</number>
                </property>
                <item>
                 <widget class="QLabel" name="label" >
                  <property name="text" >
                   <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" font-weight:600;">Welcome to InterVerdiKom!&lt;/span>&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">Please open or create a project file to start analyzing or importing data.&lt;/p>&lt;/body>&lt;/html></string>
                  </property>
                  <property name="alignment" >
                   <set>Qt::AlignCenter</set>
                  </property>
                  <property name="wordWrap" >
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
              <widget class="QWidget" name="openedProjectFilePage" >
               <layout class="QVBoxLayout" >
                <item>
                 <widget class="QLabel" name="label_3" >
                  <property name="text" >
                   <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" font-weight:600;">You just opened an existing project.&lt;/span>&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">You can select a module from the tree view on the left to inspect data or perform various actions associated with the module.&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">You can also choose to import new data using the "Import" menu.&lt;/p>&lt;/body>&lt;/html></string>
                  </property>
                  <property name="alignment" >
                   <set>Qt::AlignCenter</set>
                  </property>
                  <property name="wordWrap" >
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
              <widget class="QWidget" name="newProjectFilePage" >
               <layout class="QVBoxLayout" >
                <property name="spacing" >
                 <number>6</number>
                </property>
                <property name="margin" >
                 <number>9</number>
                </property>
                <item>
                 <widget class="QLabel" name="label_2" >
                  <property name="text" >
                   <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal; text-decoration:none;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" font-weight:600;">You just created a new project.&lt;/span>&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-weight:600;">&lt;span style=" font-weight:400;">To proceed, import data by using the "Import" menu.&lt;/span>&lt;/p>&lt;/body>&lt;/html></string>
                  </property>
                  <property name="alignment" >
                   <set>Qt::AlignCenter</set>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
              <widget class="QWidget" name="allDeletedPage" >
               <layout class="QVBoxLayout" >
                <item>
                 <widget class="QLabel" name="label_4" >
                  <property name="text" >
                   <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" font-weight:600;">All modules have been deleted.&lt;/span>&lt;/p>&lt;/body>&lt;/html></string>
                  </property>
                  <property name="alignment" >
                   <set>Qt::AlignCenter</set>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="LogTextField" name="outputConsole" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Ignored" hsizetype="Ignored" >
         <horstretch>0</horstretch>
         <verstretch>1</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize" >
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="verticalScrollBarPolicy" >
        <enum>Qt::ScrollBarAsNeeded</enum>
       </property>
       <property name="readOnly" >
        <bool>true</bool>
       </property>
       <property name="html" >
        <string>&lt;!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0//EN" "http://www.w3.org/TR/REC-html40/strict.dtd">
&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" color:#228b22;">InterVerdiKom ready.&lt;/span>&lt;/p>
&lt;p style="-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; color:#228b22;">&lt;/p>&lt;/body>&lt;/html></string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1040</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuConsole" >
    <property name="title" >
     <string>Console</string>
    </property>
    <addaction name="actionSave_Console_As" />
    <addaction name="separator" />
    <addaction name="actionClear_Console" />
   </widget>
   <widget class="QMenu" name="menuHelp" >
    <property name="title" >
     <string>Help</string>
    </property>
    <addaction name="actionAbout" />
   </widget>
   <widget class="QMenu" name="menuTools" >
    <property name="title" >
     <string>Tools</string>
    </property>
    <addaction name="actionCreate_Trace_From_Video_File" />
   </widget>
   <widget class="QMenu" name="menuFile" >
    <property name="title" >
     <string>File</string>
    </property>
    <addaction name="actionNew" />
    <addaction name="actionOpen" />
    <addaction name="separator" />
    <addaction name="actionSave" />
    <addaction name="actionSave_As" />
    <addaction name="separator" />
    <addaction name="actionQuit" />
   </widget>
   <widget class="QMenu" name="menuImport" >
    <property name="title" >
     <string>Import</string>
    </property>
    <addaction name="actionImport_Video_File" />
    <addaction name="actionImport_Trace" />
    <addaction name="actionImport_Model" />
   </widget>
   <widget class="QMenu" name="menuActions" >
    <property name="enabled" >
     <bool>false</bool>
    </property>
    <property name="title" >
     <string>Actions</string>
    </property>
   </widget>
   <addaction name="menuFile" />
   <addaction name="menuActions" />
   <addaction name="menuImport" />
   <addaction name="menuTools" />
   <addaction name="menuConsole" />
   <addaction name="menuHelp" />
  </widget>
  <action name="actionRaw_File_Parser" >
   <property name="text" >
    <string>Create Trace...</string>
   </property>
  </action>
  <action name="actionCreate_Model" >
   <property name="text" >
    <string>Create Model...</string>
   </property>
   <property name="iconText" >
    <string>Create Model...</string>
   </property>
   <property name="toolTip" >
    <string>Create Model...</string>
   </property>
  </action>
  <action name="actionOpen_Model" >
   <property name="text" >
    <string>Open Model...</string>
   </property>
  </action>
  <action name="actionSave_model" >
   <property name="text" >
    <string>Save model</string>
   </property>
  </action>
  <action name="actionSave_model_as" >
   <property name="text" >
    <string>Save model as...</string>
   </property>
  </action>
  <action name="actionSave_Console_As" >
   <property name="text" >
    <string>Save As...</string>
   </property>
  </action>
  <action name="actionClear_Console" >
   <property name="text" >
    <string>Clear</string>
   </property>
  </action>
  <action name="actionQuit" >
   <property name="text" >
    <string>Quit</string>
   </property>
  </action>
  <action name="actionBatch_Process" >
   <property name="text" >
    <string>Batch Process...</string>
   </property>
  </action>
  <action name="actionAbout" >
   <property name="text" >
    <string>About...</string>
   </property>
  </action>
  <action name="actionSave" >
   <property name="text" >
    <string>Save</string>
   </property>
  </action>
  <action name="actionSave_As" >
   <property name="text" >
    <string>Save As...</string>
   </property>
  </action>
  <action name="actionNew" >
   <property name="text" >
    <string>New</string>
   </property>
  </action>
  <action name="actionOpen" >
   <property name="text" >
    <string>Open...</string>
   </property>
  </action>
  <action name="actionSave" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="text" >
    <string>Save</string>
   </property>
  </action>
  <action name="actionSave_As" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="text" >
    <string>Save As...</string>
   </property>
   <property name="shortcut" >
    <string/>
   </property>
  </action>
  <action name="actionImport_Video_File" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="text" >
    <string>Import Video File...</string>
   </property>
  </action>
  <action name="actionImport_Trace" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="text" >
    <string>Import Trace...</string>
   </property>
  </action>
  <action name="actionImport_Model" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="text" >
    <string>Import Model...</string>
   </property>
  </action>
  <action name="actionCreate_Trace_From_Video_File" >
   <property name="text" >
    <string>Create Trace From Video File...</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LogTextField</class>
   <extends>QTextEdit</extends>
   <header>LogTextField.h</header>
  </customwidget>
  <customwidget>
   <class>NavigationPath</class>
   <extends>QWidget</extends>
   <header>NavigationPath.h</header>
  </customwidget>
  <customwidget>
   <class>ModuleList</class>
   <extends>QTreeWidget</extends>
   <header>widget/ModuleList.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
